src/data.js

```diff
@@ -10,8 +10,10 @@ export function create (cozy, doctype, attributes) {
     if (isV2) {
       attributes.docType = doctype
     }
-    const path = createPath(cozy, isV2, doctype)
-    return cozyFetchJSON(cozy, 'POST', path, attributes).then((resp) => {
+    const path = createPath(cozy, isV2, doctype, attributes._id)
```

Comment:
If the `createPath`'s 4th parameter is `undefined`, it will be set to the [default value](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters) which is `''` (empty string).

Here is the `createPath` function: https://github.com/cozy/cozy-client-js/blob/master/src/utils.js#L62